cea/technologies/network_layout/steiner_spanning_tree.py

```diff
@@ -253,6 +253,9 @@ def calc_coord_anchor(total_demand_location, nodes_df, type_network):
         field = "QH_sys_MWhyr"
     elif type_network == "DC":
         field = "QC_sys_MWhyr"
+    elif type_network == "EL":
```

Comment:
else? there should be no `elif` clause without an `else` clause... mainly because it makes me super nervous. defensive programming would suggest to raise a `ValueError` or something with the text `"Invalid value for type_network ({type_network})".format(**locals())`